environment.yml

```diff
@@ -3,4 +3,7 @@ channels:
   - conda-forge
 dependencies:
   - python=3.11
-  - pytest=8.1.1
\ No newline at end of file
+  - pytest=8.1.1
+  - pandas=2.2.1
+  - ipykernel=6.29.3 # to run the notebook locally on vscode
```

Comment:
rather use a eda.py file than eda.ipynb

Reply:
for eda its useful to have a notebook because then you can see the results and plot directly in github online without having to run. and it does not have functions or features that we use in other files